### PR TITLE
Add Cypress tests for New Pipeline page

### PR DESCRIPTION
**Pull request by davidebriani**
This PR adds basic checks in Cypress to make sure the New Pipeline page correctly handle interactions needed to fill the form and submit a new pipeline

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   },
   "scripts": {
     "start": "webpack-dev-server --hot --inline --config webpack.dev.js",
+    "start-ci": "NO_ELM_DEBUGGER=true npm run start",
```
Comment (Pavinati) [inline package.json:30]: cool trick, but this should be done in a different PR, or at least a different commit

File: webpack.dev.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ module.exports = merge(common, {
             loader: 'elm-webpack-loader',
             options: {
               verbose: true,
-              debug: true,
+              debug: !process.env.NO_ELM_DEBUGGER,
```
Comment (Pavinati) [inline webpack.dev.js:29]: i would prefer `CI_RUN` so we can make additional changes if running in the CI or not

Comment (Pavinati) [reply webpack.dev.js:29]: or create a different file just for the CI configuration that will be merged with the dev config

File: package.json (hunk 1 above)
Comment (davidebriani) [reply package.json:30]: Fair enough, I'll add a separate commit

File: webpack.dev.js (hunk 2 above)
Comment (davidebriani) [reply webpack.dev.js:29]: A different file would be too much for such a small variation. However I'll take your advice and use `CI_RUN`, much better

**Review by Pavinati — APPROVED** [review]

**Review by bettio — APPROVED** [review]
